process/kv_store.go

```diff
@@ -19,6 +22,8 @@ import (
 type procKVStore struct {
 	process
 	Options procKVStoreOptions `json:"options"`
+
+	kvStore kv.Storer
```

Comment:
i'm assuming that moving this to the struct doesn't break the Get functionality (shared across the entire app via the package), but it's worth explicitly testing if it wasn't already tested

Reply:
I tested a scenario with 2 shared consumers and it worked out well but prior to merging do a more thorough smoke test! 👍